### PR TITLE
pythonPackages.qds_sdk: 1.12.0 -> 1.15.2

### PR DESCRIPTION
**Pull request by tinahbu**
##### Motivation for this change

Add version `1.15.2` of `qds_sdk` to Nix OSS.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/qds_sdk/default.nix [hunk 1]
```diff
@@ -7,33 +7,41 @@
...
     owner = "qubole";
     repo = "qds-sdk-py";
     rev = "V${version}";
-    sha256 = "18xhvlcfki8llv7fw2r5yfk20zds3gr78b4klwm9mkvhlhwds9rx";
+    sha256 = "0xxg9s0y6fz7vb1kab4q93q7ryi71z8x6q9qspm6s506yr3mc67l";
   };
 
-  propagatedBuildInputs = [ 
+  patch = [
+    (fetchpatch {
```
Comment (kalbasit) [inline pkgs/development/python-modules/qds_sdk/default.nix:26]: Please add a comment as to why this patch is needed for this version.

**Review by kalbasit — CHANGES_REQUESTED** [review]
Please squash your commits into one. The commit title should be:

```
pythonPackages.qds_sdk: 1.12.0 -> 1.15.2
```

The same thing applies to the title of your pull request.

:point_right: https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md

**Review by jonringer — COMMENTED** [review]
for adhering to CONTRIBUTING, please use the following commit msg:

```
python3Packages.qds_sdk: 1.12.0 -> 1.15.2
```

File: pkgs/development/python-modules/qds_sdk/default.nix (hunk 1 above)
Comment (tinahbu) [reply pkgs/development/python-modules/qds_sdk/default.nix]: 👌🏼

**Comment by tinahbu** [comment]
will create a new PR with the correct commit message

**Comment by kalbasit** [comment]
@tinahbu no need to create a new PR, you can just force push to this one.

**Pull request closed without merging**